lib/commands/connect/auth.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+var api = require('./shared.js');
+let cli = require('heroku-cli-util');
+let co  = require('co');
+let http = require('http');
+
+function callbackServer() {
+  let server = http.createServer(function (request, response) {
+    response.writeHead(200, {"Content-Type": "text/html"});
+    console.log("Authorization complete");
+    let res = "<html><body><h3>Authorization complete</h3><p>You may close this window and return to the terminal to continue.</p></body></html>";
+    response.end(res);
+    process.exit();
+  });
+  server.listen(18000);
+}
+
+module.exports = {
+  topic: 'connect',
+  command: 'auth',
+  description: 'get Salesforce authorization URL',
+  help: 'return an URL to authorize a connection to a Salesforce Org',
+  flags: [
+    {name: 'callback', char: 'c', description: 'final callback URL', hasValue: true},
+    {name: 'login', char: 'l', description: 'alternate Salesforce login URL', hasValue: true},
+    {name: 'resource', description: 'specific connection resource name', hasValue: true}
+  ],
+  needsApp: true,
+  needsAuth: true,
+  run: cli.command(co.wrap(function* (context, heroku) {
+    let connection = yield api.withConnection(context, heroku);
+    callbackServer();
+
+    let url = '/api/v3/connections/' + connection.id + '/authorize_url';
+    let args = {'environment':'production', 'next': 'http://127.0.0.1:18000'};
```

Comment:
what does this do exactly? your browser will open, you'll go through auth flow, and then connect will redirect you to this localhost url which doesn't exist? if you know the region and connection id seems like you could set the `next` url to the correct dashboard endpoint?